modules/nw-ingress-sharding.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc
+// * ingress-operator.adoc
+
+[id="nw-ingress-sharding_{context}"]
+= Configuring ingress controller sharding
+
+Ingress Controllers that have permission to view all of the labels in all
+projects can select routes to admit based on the labels. This is called Ingress
+Controller sharding. This is useful when balancing incoming traffic load among a
+set of Ingress Controllers and when isolating traffic to a specific Ingress
+Controller. For example, company A goes to one Ingress Controller and company B
+to another.
+
+.Procedure
+
+. Edit the `router-internal.yaml`:
++
+----
+# cat router-internal.yaml
```

Comment:
Is this file already supposed to exist or are they supposed to create it with the following contents? Because all the "cat" command is doing is just printing the contents of the file. Maybe be a little more specific on what they're supposed to do?